### PR TITLE
Add the parameter `out` to `opq` to select the level of verbosity of the results

### PR DESCRIPTION
**Pull request by jmaspons**
I can replicate the option for `opq_osm_id`, `opq_enclosing` and `opq_around`

### CODE REVIEW

**Comment by jmaspons** [comment]
Checks fail due to a link in the documentation to `osmdata_data_frame` which is not merged yet #285

**Comment by jmaspons** [comment]
From my side, ready for the merge @mpadge 

P.S.: Don't know why there are no test coverage checks from the gh actions anymore

File: R/opq.R [hunk 1]
```diff
@@ -76,17 +88,30 @@
...
         "); out;",
         ");\n(._;>;);\nout body;"
     ) # recurse down
+    out <- try (match.arg (out))
+    if (inherits (out, "try-error")) {
+        stop ('out parameter must be "body", "tags", "meta", "skel", "tags center" or "ids".')
+    }
+    has_geometry <- out %in% c ("body", "meta", "skel")
+    if (has_geometry && out != "body") {
+        suffix <- gsub ("out( body)*;", paste0 ("out ", out, ";"), suffix)
```
Comment (mpadge) [inline R/opq.R:110]:
```suggestion
        suffix <- gsub ("out(\\sbody)*;", paste0 ("out ", out, ";"), suffix)
```

File: R/opq.R [hunk 2]
```diff
@@ -76,17 +88,30 @@
...
+    } else if (!has_geometry) {
+        suffix <- gsub ("\\);(\\n\\(\\._;>;\\);\\n| )out( body)*;", paste0 ("); out ", out, ";"), suffix)
```
Comment (mpadge) [inline R/opq.R:112]: Why not just redefine `suffix` completely?
```suggestion
        suffix <- paste0 ("); out ", out, ";")
```

**Review by mpadge — CHANGES_REQUESTED** [review]

**Review by mpadge — APPROVED** [review]

**Comment by jmaspons** [comment]
Thanks for the review. I have rewritten this part by testing the condition for `has_geometry` adding `nodes_only`, and build the suffix from scratch. This makes the tests fails because they expect that queries with nodes_only have no "body" in the string, which is the default anyway. This limit queries with `nodes_only `and out meta;. I have changed the tests

**Pull request merged**